src/main/target/CRAZYBEEF3FR/target.h

```diff
@@ -0,0 +1,119 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#if defined(CRAZYBEEF3FS)
+#define TARGET_BOARD_IDENTIFIER "CBFS"
+#define USBD_PRODUCT_STRING     "CrazyBee F3 FS"
+#else //CRAZYBEEF3FS
+#define TARGET_BOARD_IDENTIFIER "CBFR"
+#define USBD_PRODUCT_STRING     "CrazyBee F3 FR"
+#endif
+
+#define ENABLE_DSHOT_DMAR       true
+
+#define LED0_PIN                PB3
+#define USE_BEEPER 
+#define BEEPER_PIN              PC15
```

Comment:
Seems there is an issue with the Beeper Pin

![image](https://user-images.githubusercontent.com/4823043/39925853-4da39b20-54fb-11e8-8c0e-88ecca3de052.png)